allauth/socialaccount/providers/google/tests.py

```diff
@@ -204,3 +204,15 @@ def test_social_email_verification_optional(self):
         self.assertEqual(len(mail.outbox), 1)
         self.login(self.get_mocked_response(verified_email=False))
         self.assertEqual(len(mail.outbox), 1)
+
+    def test_callback_android(self):
```

Comment:
I'm not convinced that this is an adequate test since there's clearly lots of other things happening in other tests that looks like Greek to me. Any suggestions?